core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -826,8 +826,8 @@ public boolean isExternalTable() {
     return external != null && external.equalsIgnoreCase("true");
   }
 
-  public boolean isFileLevelExternalTable() {
-    String external = tableInfo.getFactTable().getTableProperties().get("_filelevelexternal");
+  public boolean isFileLevelFormat() {
+    String external = tableInfo.getFactTable().getTableProperties().get("_filelevelformat");
```

Comment:
Please raise another PR to change the table property from `_filelevelformat` to `fileformat`.
In future, CarbonTable can be any format, it can be carbonfile or parquet, orc, etc.